projects/architecture/interfaces/primitives/addresses.proto

```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
+
+message UniversalAddress {
+  uint32 scheme = 1;
+  uint32 networkId = 2;
+  bytes id = 3;
+  uint32 checksum = 4;
+}
```

Comment:
1. Why do we need so many bytes for `scheme` or `networkId`? 
2. In general, we must to have a way to serialize/deserialize these messages in a fully binary compatible way (e.g., so that we won’t different hashes or signatures). 
3. Why is the checksum a `uint32`? In other words, are we confident that 4 bytes (e.g., 4 bytes of the hash of the address) are enough?